waddrmgr/db.go

```diff
@@ -432,6 +588,81 @@ func serializeBIP0044AccountRow(encryptedPubKey,
 	return rawData
 }
 
+// fetchAllAccounts loads information about all accounts from the database.
+// The returned value is a slice of account numbers which can be used to load
+// the respective account rows.
+func fetchAllAccounts(tx walletdb.Tx) ([]uint32, error) {
```

Comment:
Please add a TODO here to deal with the fact that we will likely want to switch over to an iterator over a slice in the future.  There can be up to 2^31-1 accounts, and while it's unlikely to be a problem anytime in the near future, clearly attempting to allocate and return a slice that large won't work.